Test Failure: io.openliberty.microprofile.opentracing.internal.tck.OpentracingTCKLauncher.launchOpentracingTck

RTC 278712

```
junit.framework.AssertionFailedError: 2022-03-18-22:29:39:712 No TCK test JUnit result files were found in the results directory (/home/jazz_build/Build/jbe/build/dev/io.openliberty.microprofile.opentracing.3.0.internal_fat_tck/autoFVT/results/tck_rest-client-tck-suite/surefire-reports) which suggests the TCK tests did not run
Errors found in mvnOutput were:
### maven test output:
22:29:39,679 [ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.systemPath' for io.openliberty:io.openliberty.restfulWS30.jsonb20provider.jakarta:jar must specify an absolute path but is null @ line 157, column 25
22:29:39,680 [ERROR] The build could not read 1 project -> [Help 1]
22:29:39,680 [ERROR]   
22:29:39,681 [ERROR]   The project io.openliberty.microprofile.opentracing:tck.runner.tck:1.0-SNAPSHOT (/home/jazz_build/Build/jbe/build/dev/io.openliberty.microprofile.opentracing.3.0.internal_fat_tck/autoFVT/publish/tckRunner/tck/pom.xml) has 1 error
22:29:39,681 [ERROR]     'dependencies.dependency.systemPath' for io.openliberty:io.openliberty.restfulWS30.jsonb20provider.jakarta:jar must specify an absolute path but is null @ line 157, column 25
22:29:39,681 [ERROR] 
22:29:39,681 [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
22:29:39,681 [ERROR] Re-run Maven using the -X switch to enable full debug logging.
22:29:39,681 [ERROR] 
22:29:39,682 [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException

	at componenttest.topology.utils.MvnUtils.findJunitResultFiles(MvnUtils.java:196)
	at componenttest.topology.utils.MvnUtils.postProcessTestResults(MvnUtils.java:575)
	at componenttest.topology.utils.MvnUtils.runCleanTest(MvnUtils.java:728)
	at componenttest.topology.utils.MvnUtils.runTCKMvnCmd(MvnUtils.java:155)
	at io.openliberty.microprofile.opentracing.internal.tck.OpentracingRestClientTCKLauncher.launchOpentracingRestClientTck(OpentracingRestClientTCKLauncher.java:59)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:199)
	at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:320)
	at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:173)
```

--- a/dev/io.openliberty.microprofile.opentracing.3.0.internal_fat_tck/publish/tckRunner/tck/pom.xml
+++ b/dev/io.openliberty.microprofile.opentracing.3.0.internal_fat_tck/publish/tckRunner/tck/pom.xml
@@ -151,10 +151,10 @@
          </dependency>
          <dependency>
             <groupId>io.openliberty</groupId>
-            <artifactId>io.openliberty.restfulWS30.jsonb20provider.jakarta</artifactId>
+            <artifactId>io.openliberty.restfulWS30.jsonb20provider</artifactId>
             <version>1</version>
             <scope>system</scope>
-            <systemPath>${io.openliberty.restfulWS30.jsonb20provider.jakarta_}</systemPath>
+            <systemPath>${io.openliberty.restfulWS30.jsonb20provider_}</systemPath>
          </dependency>
          <dependency>
             <groupId>org.glassfish</groupId>

